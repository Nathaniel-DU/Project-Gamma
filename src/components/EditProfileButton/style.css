button#edit-profile-button {
    background-color: #00203FFF;
    margin: auto;
    min-width: 90%;
    cursor: pointer;
    margin-right: 0.25em;
    margin-top: 0.5em;
    /* padding: 	0.938em; */
    border: none;
    border-radius: 10px;
    color: #fefefe;
    margin-bottom: 1.3em;

}

button:hover {
    background-color: #4A4E69;
    color: #fefefe;
  }  