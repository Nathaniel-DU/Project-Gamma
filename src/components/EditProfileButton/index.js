import React, {Component} from 'react';
import {Link} from 'react-router-dom';
import './style.css';

export default class EditProfileButton extends Component {

    
    
    render(){
        return(
            <div>
                <Link to="/"><button className='blue-button' id='edit-profile-button'>Edit Profile</button></Link>
            </div>
        )
    }
}