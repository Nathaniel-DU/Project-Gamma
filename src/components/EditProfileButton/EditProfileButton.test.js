import React from 'react';
import { shallow } from 'enzyme';
import EditProfileButton from './index';

describe(`EditProfileButton`, () => {
    it(`should render correctly in "debug" mode`, () => {
        const component = shallow(<EditProfileButton debug />);
        expect(component).toMatchSnapshot();
    });
});