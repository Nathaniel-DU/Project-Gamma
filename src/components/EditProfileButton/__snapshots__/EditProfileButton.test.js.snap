// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditProfileButton should render correctly in "debug" mode 1`] = `
<div>
  <Link
    to="/edit"
  >
    <button
      className="blue-button"
      id="edit-profile-button"
    >
      Edit Profile
    </button>
  </Link>
</div>
`;
